_1327/emails/tests.py

```diff
@@ -0,0 +1,539 @@
+from datetime import datetime
+from email.message import EmailMessage
+from email.utils import make_msgid
+from io import StringIO
+from random import random
+import tempfile
+from unittest.mock import MagicMock, mock_open, patch
+
+from bs4 import BeautifulSoup
+from django.core.files.base import ContentFile
+from django.core.files.storage import FileSystemStorage
+from django.core.management import call_command
+from django.test import override_settings, TestCase
+from django.urls import reverse
+from django.utils.timezone import make_aware
+from django_webtest import WebTest
+from freezegun import freeze_time
+from model_mommy import mommy
+
+from _1327.emails import pop3_test_server, utils
+from _1327.emails.forms import QuickSearchForm, SearchForm
+from _1327.emails.models import Email
+
+
+class EmailFormTests(TestCase):
+	def test_quicksearch_form(self):
+		form = QuickSearchForm(data={})
+		self.assertFalse(form.is_valid())
+
+		form = QuickSearchForm(data={'text': 'Stollen'})
+		self.assertTrue(form.is_valid())
+
+	def test_search_form(self):
+		form = SearchForm(data={})
+		self.assertFalse(form.is_valid())
+
+		form = SearchForm(data={'text': 'Stollen'})
+		self.assertTrue(form.is_valid())
+		self.assertFalse(form.cleaned_data['has_attachments'])
+
+		form = SearchForm(data={'sender': 'Santa Claus'})
+		self.assertTrue(form.is_valid())
+
+		form = SearchForm(data={'receiver': 'Santa Claus'})
+		self.assertTrue(form.is_valid())
+
+		form = SearchForm(data={'received_before': '2018-12-24'})
+		self.assertTrue(form.is_valid())
+
+		form = SearchForm(data={'received_after': '2018-12-24'})
+		self.assertTrue(form.is_valid())
+
+		form = SearchForm(data={'has_attachments': True})
+		self.assertTrue(form.is_valid())
+
+
+class EmailModelTest(TestCase):
+	def test_to(self):
+		email = Email(to_names=[], to_addresses=[])
```

Comment:
is it possible to create these with `mommy.make(Email, ...)`? that would make sure all other fields are also filled and the objects works as a whole.